demostat/templates/demostat/demo_detail.html

```diff
@@ -52,10 +58,10 @@ <h5 class="card-title">Was?</h5>
       <div class="clickoverlay embed-responsive-item">
         <p class="text-center">Klicken um die Karte freizuschalten</p>
       </div>
-      <iframe class="embed-responsive-item" src="https://www.openstreetmap.org/export/embed.html?bbox={{ demo.location.box_left }}%2C{{ demo.location.box_bottom }}%2C{{ demo.location.box_right }}%2C{{ demo.location.box_top }}&amp;layer=mapnik&amp;marker={{ demo.location.marker_lat }}%2C{{ demo.location.marker_lon }}"></iframe>
+      <div id="mapid" class="embed-responsive-item"></div>
```

Comment:
Dieser Name ist nicht sonderlich Aussagekräftig.
```suggestion
      <div id="location-map" class="embed-responsive-item"></div>
```